### PR TITLE
Feature/jira 1 add h2 database

### PR DESCRIPTION
**Pull request by ktweel**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
 		<sonar.exclusions>**/ProtobufMessages.java</sonar.exclusions>
 		<sonar.coverage.exclusions>**/FuseWebRequestTraceFilter.java,**/SettlementMission.java</sonar.coverage.exclusions>
 		<!-- Allow us to skip Unit Tests only, if need be, with -Dskip.surefire.tests, but retain the skip all tests functionality of -DskipTests -->
-		<skip.surefire.tests>${skipTests}</skip.surefire.tests>
+		<!--<skip.surefire.tests>${skipTests}</skip.surefire.tests>-->
```
Comment (wmvsilva) [inline pom.xml]: Is this supposed to be commented out?

File: pom.xml [hunk 2]
```diff
@@ -152,6 +152,7 @@
 		<dependency>
 			<groupId>com.h2database</groupId>
 			<artifactId>h2</artifactId>
+			<!--<version>1.4.196</version>-->
```
Comment (wmvsilva) [inline pom.xml]: You should have a specific version for this.

> It's worth emphasising that any approach that allows Maven to pick the dependency versions (LATEST, RELEASE, and version ranges) can leave you open to build time issues, as later versions can have different behaviour (for example the dependency plugin has previously switched a default value from true to false, with confusing results).

https://stackoverflow.com/questions/30571/how-do-i-tell-maven-to-use-the-latest-version-of-a-dependency/1172371#1172371

File: src/main/java/org/galatea/starter/domain/AlphaVantageMetaData.java [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
-    @JsonProperty("2. Symbol")
-    private String symbol;
-    @JsonProperty("3. Last Refreshed")
-    private String lastRefreshed;
-    @JsonProperty("4. Output Size")
-    private String outputSize;
-    @JsonProperty("5. Time Zone")
-    private String timeZone;
+  @JsonProperty("1. Information")
+  private String information;
```
Comment (wmvsilva) [inline src/main/java/org/galatea/starter/domain/AlphaVantageMetaData.java]: Could you add some spacing in-between these so it easier to read?

File: src/main/java/org/galatea/starter/domain/StockData.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+@Data
+@Table(name="stock")
+public class StockData {
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private Long id;
+  @NonNull
+  @Column(name="stockSymbol")
+  private String stockSymbol;
+//  @NonNull
```
Comment (wmvsilva) [inline src/main/java/org/galatea/starter/domain/StockData.java]: Is this supposed to be commented out?

**Review by wmvsilva — CHANGES_REQUESTED** [review]
Just a few quick suggestions / questions before I head out today.

File: src/main/java/org/galatea/starter/domain/AlphaVantageMetaData.java (hunk 3 above)
Comment (ktweel) [reply src/main/java/org/galatea/starter/domain/AlphaVantageMetaData.java]: done

File: pom.xml (hunk 2 above)
Comment (ktweel) [reply pom.xml]: fixed

File: src/main/java/org/galatea/starter/domain/StockData.java (hunk 4 above)
Comment (ktweel) [reply src/main/java/org/galatea/starter/domain/StockData.java]: nope, thanks for the catch

File: src/main/java/org/galatea/starter/service/StockPriceService.java [hunk 5]
```diff
@@ -6,59 +6,140 @@
...
+@Component
 public class StockPriceService {
 
-  public static String getPriceData(String symbol, int days) {
-    final String uri = "https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=%s&outputsize=%s&apikey=PFPOE75HO1WCKW7H";
+  @Autowired
+  StockDataRepository repository;
+  @Autowired
+  ObjectMapper mapper;
+  private final String uri = "https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=%s&outputsize=%s&apikey=PFPOE75HO1WCKW7H";
```
Comment (GalaFirsh) [inline src/main/java/org/galatea/starter/service/StockPriceService.java]: let's make this a property in our yaml that we inject in!

**Pull request merged**